spec/components_spec.rb

```diff
@@ -355,8 +355,8 @@
 
 RSpec.describe "raw" do
   it 'creates a wrapper that ignores anything inside' do
-    input = "<raw><<LCG Program\TG LCG Coupon Code Default='246996'>></raw>"
-    expected = "<<LCG Program\TG LCG Coupon Code Default='246996'>>"
+    input = "<body><raw><<LCG Program\TG LCG Coupon Code Default='246996'>></raw></body>"
+    expected = "<?xml version=\"1.0\"?>\n<body><<LCG ProgramTG LCG Coupon Code Default='246996'>></body>\n"
```

Comment:
To be clear: this change is fine for this PR, I'll just want to tackle that change as well to fix the other bug and then reconsider this.